Pratiksha/html/Form/form.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="form.css">
+</head>
+<body>
+   <div class="main">
+       <h1 >MI-1o Intern Assessment</h1>
```

Comment:
MI-1o Intern Assessment, Personal Details and Academic Details in sab ko aap div tag ke undar h2 aur h3 ke under rakhiye ye proper way ho ga

Reply:
ok